cdap-validation/bin/testing.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/python
+
+import test_helpers as helpers
+import json
+import ambari
+import cm
+
+##### TEST FRAMEWORK FUNCTIONS #####
+
+input_json = "input.json"
```

Comment:
Comment JSON layout, so someone can write this file to duplicate/test this code.